extensions/replication/queueProcessor/task.js

```diff
@@ -1,6 +1,5 @@
 'use strict'; // eslint-disable-line
 const assert = require('assert');
-const werelogs = require('werelogs');
```

Comment:
Was this change intended to be committed?

Reply:
Yes, I noticed it was not used in the file. I guess maybe it is outside the scope of this PR though.

Reply:
Removed these changes.